homeassistant/components/weather/__init__.py

```diff
@@ -1088,6 +1111,51 @@ async def async_get_forecast_service(
     }
 
 
+async def async_get_forecast_value_service(
+    weather: WeatherEntity, service_call: ServiceCall
+) -> ServiceResponse:
+    """Get weather forecast for specified attribute."""
+    forecast_attribute = service_call.data.get("attribute")
```

Comment:
The schema marks this as required:
```suggestion
    forecast_attribute = service_call.data.["attribute"]
```